Enhance troubleshooting guide for NATS issues (#6307)
SkylineCommunications/dataminer-docs: dataminer/Troubleshooting/Procedures/Investigating_NATS_Issues.md

- **Authorization violations** indicate credential mismatches or missing credential files.

  To resolve this issue, [Repair the NATS cluster](#resettingrepairing-the-brokergateway-nats-cluster).

- **Connection refused errors** suggest firewall or antivirus issues. This can also mean that the NATS service is not running.

>   To resolve this issue, check the firewall rules, network connectivity, and the nats-server process if it is running.

Reviewer: This isn't entirely clear. Is this what you meant?

```suggestion
  To resolve this issue, check the firewall rules, the network connectivity, and whether the nats-server process is running.
```
